### PR TITLE
Con-332 PSR Download Via Browser Print

### PR DESCRIPTION
**Pull request by Ashley-Bass**
JS dependent code that allows for downloading of PSR by utilising a print style sheet and the browsers print functionality

### CODE REVIEW

File: src/SFA.DAS.PSRService.Web/wwwroot/assets/stylesheets/esfa-govuk-base.css [hunk 1]
```diff
@@ -742,6 +746,81 @@ hr {
       background-image: url("../images/icon-pointer-2x.png");
       background-size: 30px 19px; } }
 
+.button-secondary {
```
Comment (cofaulco) [inline src/SFA.DAS.PSRService.Web/wwwroot/assets/stylesheets/esfa-govuk-base.css]: I think these changes should be made in the site.css in wwwroot/css

File: src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml [hunk 2]
```diff
@@ -23,98 +27,98 @@
...
+                <legend>
+                    <h1 class="heading-xlarge">Submitted details</h1>
+                </legend>
+            }
+            else
+            {
+                if (Model.IsReadOnly)
                 {
                     <legend>
-                        <h1 class="heading-xlarge">Submitted details</h1>
```
Comment (cofaulco) [inline src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml]: what does this change have to do with printing the page?

File: src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml [hunk 3]
```diff
@@ -23,98 +27,98 @@
...
-                    {
-                        <a class="link-back" asp-controller="Home" asp-action="Index">Back</a>
-                    }
-
+                    <a class="link-back" asp-controller="Report" asp-action="Edit">Back</a>
                 }
                 else
                 {
-                    <a class="link-back" asp-controller="Report" asp-action="List">Back</a>
+                    <a class="link-back" asp-controller="Home" asp-action="Index">Back</a>
```
Comment (cofaulco) [inline src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml]: what does this change have to do with printing the page?

File: src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml [hunk 4]
```diff
@@ -23,98 +27,98 @@
...
                 {
                     <legend>
-                        <h1 class="heading-xlarge">Submitted details</h1>
+                        <h1 class="heading-xlarge">View details</h1>
                     </legend>
+                    <p style="white-space: pre-line">@Model.Subtitle</p>
                 }
                 else
                 {
-                    if (Model.IsReadOnly)
```
Comment (cofaulco) [inline src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml]: what does this change have to do with printing the page?

File: src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml [hunk 5]
```diff
@@ -23,98 +27,98 @@
...
-                            @(Model.IsReadOnly ? "These are the current errors" : "Please fix these errors")
-                        </h2>
-                    </div>
+                    <legend>
+                        <h1 class="heading-xlarge">Review details</h1>
+                    </legend>
+                    <p style="white-space: pre-line">@Model.Subtitle</p>
                 }
-
-                <div class="form-group">
```
Comment (cofaulco) [inline src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml]: what does this change have to do with printing the page?

File: src/SFA.DAS.PSRService.Web/wwwroot/assets/stylesheets/esfa-govuk-base.css (hunk 1 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.PSRService.Web/wwwroot/assets/stylesheets/esfa-govuk-base.css]: Done

File: src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml (hunk 2 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml]: Github weirdness no changes made

File: src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml (hunk 3 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml]: Github weirdness no changes made

File: src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml (hunk 4 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml]: Github weirdness no changes made

File: src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml (hunk 5 above)
Comment (Ashley-Bass) [reply src/SFA.DAS.PSRService.Web/Views/Report/Summary.cshtml]: Github weirdness no changes made

**Pull request closed without merging**